packages/mattermost-redux/src/selectors/entities/channels.ts

```diff
@@ -336,6 +336,20 @@ export const getChannelSetInCurrentTeam: (state: GlobalState) => string[] = crea
     },
 );
 
+export const getChannelSetForAllTeam: (state: GlobalState) => string[] = createSelector(
```

Comment:
nit: is the naming correct? It seems as it it should be `getChannelSetForAllTeams`, or am I misreading something?